### ISSUE TITLE
 871 paths removed from GEDCOM 5.5.1  - limited backward compatability.

### ISSUE THREAD

**Issue by augean** [FamilySearch/GEDCOM#425]
stillthere = 2188
removed = 871    
Almost a 1/3 of  GEDCOM 5  paths are removed in GEDCOM 7

Attached is a list of paths that have been removed,  and paths that are still there.

 [stillthere.txt](https://github.com/FamilySearch/GEDCOM/files/13999418/stillthere.txt)
 [removed.txt](https://github.com/FamilySearch/GEDCOM/files/13999419/removed.txt)

looks like there is very limited backward compatibility
most GEDCOM 5.5.1 files can't be used in the GEDCOM7 schema.

GEDCOM 7 files will have to be rebuilt against the new schema
the conversion would be very complex from GEDCOM 5 to GEDCOM7.

Any suggestions ?
Most existing files will break.

**Comment by tychonievich** [comment]
You are correct; 7.0 is a major (not minor or patch) version, and hence not backwards compatible with earlier versions. A few of the removed paths are removed entirely (e.g. SUBN), but most are refactored into another structure (e.g. FONE → TRAN). We have a guide to conversion from 5.5.1 at https://gedcom.io/migrate/ and a few community-supplied example converters at https://gedcom.io/tools/#v551-to-v70-conversion-code

FYI, saying "GEDCOM 5" is technically vague; 5.5.1, 5.5, 5.4, 5.3, and 5.0 were not backwards compatible either. We do not currently have a guide for conversion from 5.5 or earlier.

**Comment by augean** [comment]
its   actually    Release 5.5.1      I used for the comparison
